spec/integration/categories_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe 'Authentication', type: :feature do
+  let(:user) { User.new(name: 'Ashna', email: 'ashnaalipk@gmail.com', password: 'password') }
+end
+
+RSpec.describe 'Authentication', type: :request do
+  let(:user) { User.new(name: 'Ashna', email: 'ashnaalipk@gmail.com', password: 'password') }
+
+  before do
+    sign_in user
+  end
+end
+
+RSpec.describe 'User', type: :feature do
+  let(:user) { FactoryBot.create(:user) }
+
+  before do
+    sign_in user
+  end
+end
```

Comment:
- [ ] You have not written any integration tests here that test the functionality of your categories. Please try to write robust integration tests here 🙏

Reply:
fixed:
![image](https://github.com/Ashnaali3255/Budget-App-rails/assets/116655378/eef2a140-f7b6-46a2-add3-99554d255518)